python/paddle/tensor/random.py

```diff
@@ -401,6 +415,13 @@ def uniform(shape, dtype='float32', min=-1.0, max=1.0, seed=0, name=None):
 
 
     """
+    if dtype is None:
+        dtype = paddle.framework.get_default_dtype()
+        if dtype not in ['float32', 'float64']:
+            raise TypeError(
+                "uniform only supports [float32, float64], but the default dtype is %s"
+                % dtype)
+
```

Comment:
Not important, but I want to suggest you to abstract and reuse the same code, try to reduce duplicated code (maybe you already know).

Reply:
Done, refer to PR #26951